qutebrowser/browser/commands.py

```diff
@@ -1748,3 +1756,19 @@ def fullscreen(self, leave=False, enter=False):
 
         log.misc.debug('state before fullscreen: {}'.format(
             debug.qflags_key(Qt, window.state_before_fullscreen)))
+
+    def possibly_open_in_existing_tab(self, url):
```

Comment:
I see various issues with this function:

- It doesn't really belong here - it's not a command, and it's also used from outside `commands.py`. Why not move it to `tabsfinder.py`? Or maybe in `mainwindow.py` seeing that it already has various window utility functions? Maybe it'd even make sense to add a new `tabutils.py` instead of `tabsfinder.py`?
- Its name doesn't convey that it switches to a tab. It sounds like it'd check if an URL is possibly open in another tab, but it does more than that. Maybe a name like `switch_to_existing_tab` or so woudl be better?

Reply:
Moved and renamed.